programmers/20210506/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+    let answer = [];
+    for (let i = 0; i < numbers.length; i++) {
+        for (let j = 1; j < numbers.length && i !== j; j++) {
+            let sumNum = numbers[i] + numbers[j];
+            let repeat = 0;
+            for (let k = 0; k < answer.length; k++) {
+                if (sumNum === answer[k]) {
+                    repeat++;
+                }
+            }
+            if (repeat === 0) {
+                answer.push(sumNum);
+            }
+        }
+    }
+    return answer.sort((a, b) => a - b);
+}
```

Comment:
지금도 너무 잘푸셔서 딱히 알려드릴게 없지만
중복을 제거하기위해서 for문을 하나 더 쓰신거 같아요~
자바스크립트에서는 Set을 이용해서 중복을 제거할 수 있습니다.
Set에 대해서 알아보시면 다음에는 더 잘 푸실수 있을꺼 같습니다!

```suggestion
function solution(numbers) {
    const answer = [];

    for(let i = 0; i < numbers.length; i++){
        for(let j= i+1; j < numbers.length; j++){
            answer.push(numbers[i] + numbers[j]);            
        }
    }
    
    const set = Array.from(new Set(answer));
    
    return set.sort((a, b) => { return a - b });
}
```